cibyl/sources/zuul/query.py

```diff
@@ -272,6 +272,35 @@ def _handle_jobs_query(zuul, **kwargs):
     return model
 
 
+def log_query_conditions(**kwargs):
+    """Logs any special conditions the query is going to take against the
+    arguments. Some of these are, for example, adding an argument that has
+    been omitted by the user.
+
+    :param kwargs: The arguments to check.
+    """
+    if 'tenants' not in kwargs:
+        LOG.info(
```

Comment:
I wonder if it wouldn't be better to log this in debug level. It's not a lot of lines but if the user were to query multiple systems it would be a lot of redundant information to show in info IMO